### PR TITLE
[doc]show statements

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
### What changes were proposed in this pull request?

Reorganized show statements doc.

### CODE REVIEW

File: docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-charset-syntax.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+`SHOW CHARSET` 返回所有可用的字符集。目前支持两种类型：utf8、utf8mb4。其中默认字符集为 utf8。**Nebula Graph** 将 utf8 进行了扩展，utf8 同时支持 4 个字节的字符，因此，utf8 和 utf8mb4 是等价的。
+
+```ngql
+nebula> SHOW CHARSET;
+========================================================
+| Charset | Description   | Default collation | Maxlen |
+========================================================
+| utf8    | UTF-8 Unicode | utf8_bin          | 4      |
+--------------------------------------------------------
+```
```
Comment (panda-sheep) [inline docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-charset-syntax.md]: The results of` show charset and show collation ` is incomplete.
when pr  1782 merged, you need update it.

Comment (Amber1990Zhang) [reply docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-charset-syntax.md]: thanks for your kind remind

File: docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-collation-syntax.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# SHOW COLLATION 语法
+
+```ngql
+SHOW COLLATION
+```
+
+`SHOW COLLATION` 语句列出服务器支持的排序规则。目前支持四种类型：utf8_bin、utf8_general_ci、utf8mb4_bin、utf8mb4_general_ci。字符集为 utf8 时，默认 collate 为 utf8_bin；字符集为 utf8mb4 时，默认 collate 为 utf8mb4_bin。utf8_general_ci 和 utf8mb4_general_ci 都是忽略大小写的比较，行为同 MySQL 一致。
+
```
Comment (panda-sheep) [inline docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-collation-syntax.md]: `SHOW COLLATION` 语句列出Nebula Graph 目前支持的所有排序规则。目前支持四种排序规则：

File: docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-charset-syntax.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# SHOW CHARSET 语法
+
+```ngql
+SHOW CHARSET
+```
+
+`SHOW CHARSET` 返回所有可用的字符集。目前支持两种类型：utf8、utf8mb4。其中默认字符集为 utf8。**Nebula Graph** 将 utf8 进行了扩展，utf8 同时支持 4 个字节的字符，因此，utf8 和 utf8mb4 是等价的。
+
```
Comment (panda-sheep) [inline docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-charset-syntax.md]: `SHOW CHARSET`  语句列出Nebula Graph 目前支持的所有字符集。目前支持两种字符集：

Comment (Amber1990Zhang) [reply docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-charset-syntax.md]: I think it's ok here.

**Comment by panda-sheep** [comment]
The part of charset and collation is ok.

File: docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-indexes-syntax.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+# SHOW INDEXES 语法
+
+```ngql
+SHOW {TAG | EDGE} INDEXES
+```
+
+`SHOW INDEXES` 用于列出已创建完成的标签或边类型的索引信息。`SHOW INDEXES` 返回以下字段：索引 ID 和 索引名称。
```
Comment (laura-ding) [inline docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-indexes-syntax.md]: add example

File: docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-roles-syntax.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# SHOW ROLES 语法
+
+```ngql
+SHOW ROLES IN <space_name>>
+```
+
+`SHOW ROLES` 语句显示分配给用户账户的角色。`SHOW ROLES` 输出以下列：用户账户和角色类型。
```
Comment (laura-ding) [inline docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-roles-syntax.md]: ditto

File: docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-snapshots-syntax.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# SHOW SNAPSHOTS 语法
+
+```ngql
+SHOW SNAPSHOTS
```
Comment (laura-ding) [inline docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-snapshots-syntax.md]: ditto

File: docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-users-syntax.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# SHOW USERS 语法
+
+```ngql
+SHOW USERS
+```
+
+`SHOW USERS` 语句显示用户信息。 `SHOW USERS` 输出以下列：账户名。
```
Comment (laura-ding) [inline docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-users-syntax.md]: ditto

File: docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-indexes-syntax.md (hunk 4 above)
Comment (Amber1990Zhang) [reply docs/manual-CN/2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-indexes-syntax.md]: I'm migrating docs to another [repo](https://github.com/vesoft-inc/nebula-docs). Will fix there.

**Pull request merged**